src/util/integerfactoring.c

```diff
@@ -698,22 +1253,145 @@ static uint64_t gaIFLGetSmallestFactorv(int n, const ga_factor_list* fl, int* id
 	return hasFactors ? f : 1;
 }
 
+int      gaIFLsprintf(char* str, const ga_factor_list* fl){
```

Comment:
I already changed that function to use strb_appendf instead.  snprintf is available on MSVC, but you need to to contortions to get it which I'd rather not replicate all over the place.

If you want to print a factor list for debugging you can use either strb_cstr(), followed by the usual printf() or `strb_dump(strb *, FILE *)` to write the content of the strb to the specified FILE handle.